data/data/bootstrap/files/usr/local/bin/bootstrap-cluster-gather.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+
+function queue() {
+    local TARGET="${ARTIFACTS_TEMP}/${1}"
+    shift
+    # shellcheck disable=SC2155
+    local LIVE="$(jobs | wc -l)"
+    while [[ "${LIVE}" -ge 45 ]]; do
+        sleep 1
+        LIVE="$(jobs | wc -l)"
+    done
+    # echo "${@}"
+    if [[ -n "${FILTER-}" ]]; then
+        # shellcheck disable=SC2024
+        sudo KUBECONFIG="${GATHER_KUBECONFIG}" "${@}" | "${FILTER}" >"${TARGET}" &
+    else
+        # shellcheck disable=SC2024
+        sudo KUBECONFIG="${GATHER_KUBECONFIG}" "${@}" >"${TARGET}" &
+    fi
+}
+
+function cluster_bootstrap_gather() {
+    GATHER_KUBECONFIG="/opt/openshift/auth/kubeconfig"
+
+    ALTERNATIVE_KUBECONFIG="/etc/kubernetes/bootstrap-secrets/kubeconfig"
+    if [[ -f ${ALTERNATIVE_KUBECONFIG} ]]; then
+        GATHER_KUBECONFIG=${ALTERNATIVE_KUBECONFIG}
+    fi
+
+    echo "Using ${GATHER_KUBECONFIG} as KUBECONFIG"
+
+    ARTIFACTS_TEMP="/tmp/artifacts-api-server-temp-$(date +%s%N)"
```

Comment:
Let's use a real temp directory instead of relying on timestamp.
```suggestion
    ARTIFACTS_TEMP="$(mktemp -d)"
```